### PR TITLE
AnnaTereliusweatherappproject6

### PR DESCRIPTION
**Pull request by AnnaTerelius**

### CODE REVIEW

**Comment by ericaglimsholt** [comment]
Please add a Netlify link to you PR

**Comment by AnnaTerelius** [comment]
Netlify link: annatereliusproject6weatherapp

File: script.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+const dayNames = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"]
+
+fetch("https://api.openweathermap.org/data/2.5/weather?q=Stockholm,Sweden&units=metric&APPID=f4893e301384d5deeafa555ccaa61aaa")
+.then((response) => {
+  return response.json()
+})
+.then((json) => {
+  console.log(json)
+  weatherContainer.innerHTML = `<h1> ${json.name} </h1>` 
+  tempContainer.innerHTML = `<p>${json.main.temp.toFixed(1)} &#176;</p><p>${json.weather[0].description}</p>`
```
Comment (ericaglimsholt) [inline script.js]: Nice of you using `toFixed()`!

File: script.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  sunriseContainer.innerHTML = `<p>Sunrise ${sunriseTime} </p>`
+  sunsetContainer.innerHTML = `<p>Sunset ${sunsetTime} </p>`
+})
+
+fetch("https://api.openweathermap.org/data/2.5/forecast?q=Stockholm,Sweden&units=metric&APPID=f4893e301384d5deeafa555ccaa61aaa")
+.then((response) => {
+  return response.json()
+})
+.then((json) =>{
+  console.log(json)
```
Comment (ericaglimsholt) [inline script.js]: Delete log

File: script.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  let forcast = "" 
+  json.list.forEach((element) =>{
+    let weatherDate = new Date(element.dt * 1000)
+    console.log(weatherDate.getHours())
```
Comment (ericaglimsholt) [inline script.js]: Delete log

File: script.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    if (weatherDate.getHours() === 13){
+      forcast += `<div class="forcastDay">${dayNames[weatherDate.getDay()]}:</div>`
+      forcast += `<div class="forcastTemp">${element.main.temp.toFixed(1)} &#176;</div>`
+      forcast += `<div class="forcastDescription">${element.weather[0].description}</div>`
+      console.log(dayNames[weatherDate.getDay()])
```
Comment (ericaglimsholt) [inline script.js]: Delete log

File: script.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    }
+  });
+  forcastContainer.innerHTML = forcast
+})
+// container2.innerHTML = `<p>Wed: ${json.list[8].main.temp.toFixed(1)} &#176; ${json.list[8].weather[0].description}</p> <p> Thu: ${json.list[16].main.temp.toFixed(1)} &#176; ${json.list[16
+// ].weather[0].description}</p> <p> Fri: ${json.list[24].main.temp.toFixed(1)} &#176; ${json.list[24].weather[0].description}</p> <p> Sat: ${json.list[32].main.temp.toFixed(1)} &#176; ${json.list[32].weather[0].description}</p>`
```
Comment (ericaglimsholt) [inline script.js]: Delete comments

File: script.js [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+    }
+  });
+  forcastContainer.innerHTML = forcast
+})
+// container2.innerHTML = `<p>Wed: ${json.list[8].main.temp.toFixed(1)} &#176; ${json.list[8].weather[0].description}</p> <p> Thu: ${json.list[16].main.temp.toFixed(1)} &#176; ${json.list[16
+// ].weather[0].description}</p> <p> Fri: ${json.list[24].main.temp.toFixed(1)} &#176; ${json.list[24].weather[0].description}</p> <p> Sat: ${json.list[32].main.temp.toFixed(1)} &#176; ${json.list[32].weather[0].description}</p>` 
+  
+
+
+
```
Comment (ericaglimsholt) [inline script.js]: Delete empty rows

File: style.css [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+* {
+    font-family: 'McLaren', cursive;
+}
+
+.backgroundContainer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: left;
+    background-size: cover;
```
Comment (ericaglimsholt) [inline style.css]: I figure that you want the background to all over the page with this? And it's how you use it. The problem is that your body hasn't to much height. You can change that by adding `height: 100vh;` vh stands for viewport height

File: style.css [hunk 8]
```diff
@@ -0,0 +1,107 @@
+body{
```
Comment (ericaglimsholt) [inline style.css]:
```suggestion
body {
```

File: style.css [hunk 9]
```diff
@@ -0,0 +1,107 @@
+body{
+    margin:0px;
```
Comment (ericaglimsholt) [inline style.css]:
```suggestion
    margin: 0px;
```

File: style.css [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+    margin-top: 50px;
+    margin-bottom: 50px;
+    background-color: white;
+    padding: 30px;
+    text-align: center;
+    border-radius: 20px;
+    width: 70%;
+}
+
+h1{
```
Comment (ericaglimsholt) [inline style.css]:
```suggestion
h1 {
```

File: style.css [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+.other{
+    background-image: linear-gradient(rgb(101, 179, 243), rgb(185, 201, 206));
+}
+
+.allWeather{
+    border: 2px solid black;
+    height: 450px;
+    margin-top: 50px;
+    margin-bottom: 50px;
```
Comment (ericaglimsholt) [inline style.css]: Collide these two to one row `margin: 50px 0;`

**Comment by AnnaTerelius** [comment]
Thank you Erica for the comments, very valuable for me! I Will try to nail it next time! / Anna

Skickat från min iPhone

3 nov. 2019 kl. 20:10 skrev Erica Glimsholt <notifications@github.com<mailto:notifications@github.com>>:


@ericaglimsholt approved this pull request.

Good work Anna! 🚀
I've added a couple of comments for you to read through. The major thing is that I think you need to take 30 minutes once you are done coding and just go trough everything. Go through everything so it looks the same everywhere (like spacing), no console.logs (they are only used for your own purpose) and leave comments that explains your code. With comments I mean not everywhere but in those places where it maybe isn't easy to understand what's going. I would also like that you write a description to your PR (you get that option when you're creating a PR) where you maybe write somethings (if you have questions, what was hard and so on) and adding you Netlify link which is required :)

________________________________

In index.HTML<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866429&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098898017&sdata=2yroWMFtlyO0bk6fJNABhGVFf4zflb9mkMRPpmPFN9I%3D&reserved=0>:

> @@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=McLaren&display=swap" rel="stylesheet">
+    <title>weatherApp</title>
+</head>
+<body>
+    <div id="backgroundContainer" class="backgroundContainer">
+        <div class = allWeather>


⬇️ Suggested change

-        <div class = allWeather>
+        <div class="allWeather">


________________________________

In index.HTML<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866434&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098898017&sdata=ThDQUKcHYp1UpBlY%2BHo1%2F57tqZqis7r1skwW%2FYMJsHs%3D&reserved=0>:

> +<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=McLaren&display=swap" rel="stylesheet">
+    <title>weatherApp</title>
+</head>
+<body>
+    <div id="backgroundContainer" class="backgroundContainer">
+        <div class = allWeather>
+            
+            <div id = "weather" class="weather">
+            Loading...


Indent

________________________________

In index.HTML<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866438&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098908026&sdata=HebkLpEceOUvJI6RS5QL96bGcj9%2BMYukvhX6O3Tl29M%3D&reserved=0>:

> @@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=McLaren&display=swap" rel="stylesheet">
+    <title>weatherApp</title>
+</head>
+<body>
+    <div id="backgroundContainer" class="backgroundContainer">
+        <div class = allWeather>
+            
+            <div id = "weather" class="weather">


⬇️ Suggested change

-            <div id = "weather" class="weather">
+            <div id="weather" class="weather">


________________________________

In index.HTML<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866445&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098908026&sdata=kXctdyhhRBwwxZMfYWLhxnT1wTGkkpELkXDJiVxvT3k%3D&reserved=0>:

> +    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=McLaren&display=swap" rel="stylesheet">
+    <title>weatherApp</title>
+</head>
+<body>
+    <div id="backgroundContainer" class="backgroundContainer">
+        <div class = allWeather>
+            
+            <div id = "weather" class="weather">
+            Loading...
+            </div>
+            <div class="toControlSunriseandset">
+                <p id = "sunrise" class="sunrise"></p>


⬇️ Suggested change

-                <p id = "sunrise" class="sunrise"></p>
+                <p id="sunrise" class="sunrise"></p>


________________________________

In index.HTML<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866448&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098918039&sdata=iGhMBPGnD1hAMxD2C8lY0GVUT059EcJQ6EOuVGFkMDc%3D&reserved=0>:

> +    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=McLaren&display=swap" rel="stylesheet">
+    <title>weatherApp</title>
+</head>
+<body>
+    <div id="backgroundContainer" class="backgroundContainer">
+        <div class = allWeather>
+            
+            <div id = "weather" class="weather">
+            Loading...
+            </div>
+            <div class="toControlSunriseandset">
+                <p id = "sunrise" class="sunrise"></p>
+                <p id = "sunset" class="sunset"></p>


⬇️ Suggested change

-                <p id = "sunset" class="sunset"></p>
+                <p id="sunset" class="sunset"></p>


________________________________

In index.HTML<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866454&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098918039&sdata=8eElJOC%2FJ%2B72KbkUvU6CTpZK6znXszd7EhERrjMeDSY%3D&reserved=0>:

> +    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=McLaren&display=swap" rel="stylesheet">
+    <title>weatherApp</title>
+</head>
+<body>
+    <div id="backgroundContainer" class="backgroundContainer">
+        <div class = allWeather>
+            
+            <div id = "weather" class="weather">
+            Loading...
+            </div>
+            <div class="toControlSunriseandset">
+                <p id = "sunrise" class="sunrise"></p>
+                <p id = "sunset" class="sunset"></p>
+            </div>
+            <p id = "temp" class="temp"></p>


⬇️ Suggested change

-            <p id = "temp" class="temp"></p>
+            <p id="temp" class="temp"></p>


________________________________

In index.HTML<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866464&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098928038&sdata=p831hDr%2Bas4AoSHqw0G4Q7KgnrRpvfsMj4I%2BzbRHJjw%3D&reserved=0>:

> +    <link href="https://fonts.googleapis.com/css?family=McLaren&display=swap" rel="stylesheet">
+    <title>weatherApp</title>
+</head>
+<body>
+    <div id="backgroundContainer" class="backgroundContainer">
+        <div class = allWeather>
+            
+            <div id = "weather" class="weather">
+            Loading...
+            </div>
+            <div class="toControlSunriseandset">
+                <p id = "sunrise" class="sunrise"></p>
+                <p id = "sunset" class="sunset"></p>
+            </div>
+            <p id = "temp" class="temp"></p>
+            <div id = "forcast" class="forcast">


⬇️ Suggested change

-            <div id = "forcast" class="forcast">
+            <div id="forcast" class="forcast">


________________________________

In script.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866493&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098928038&sdata=TeVkxE677D%2ByGsL%2FKpP5B3ykLD7NQBqtjKE31UKhtGA%3D&reserved=0>:

> @@ -0,0 +1,75 @@
+const weatherContainer = document.getElementById("weather")
+const tempContainer = document.getElementById("temp")


Temp? Does this stand for temperatur? Temporary? Maybe need a better name here so there's no confusing

________________________________

In script.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866504&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098938047&sdata=ou%2Bzqdhp0B90ZaBXE7sTHPHySJ6q%2BXLPUyyhiPE6Ytc%3D&reserved=0>:

> @@ -0,0 +1,75 @@
+const weatherContainer = document.getElementById("weather")
+const tempContainer = document.getElementById("temp")
+const sunriseContainer = document.getElementById("sunrise")
+const sunsetContainer = document.getElementById("sunset")
+const forcastContainer = document.getElementById("forcast")
+const dayNames = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"]
+
+fetch("https://api.openweathermap.org/data/2.5/weather?q=Stockholm,Sweden&units=metric&APPID=f4893e301384d5deeafa555ccaa61aaa")
+.then((response) => {
+  return response.json()
+})
+.then((json) => {
+  console.log(json)


Delete log

________________________________

In script.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866511&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098938047&sdata=l8yw0aYooE4hnYoBPBV9Py8RsXKKjasFmZSxcUFg2PU%3D&reserved=0>:

> +const sunriseContainer = document.getElementById("sunrise")
+const sunsetContainer = document.getElementById("sunset")
+const forcastContainer = document.getElementById("forcast")
+const dayNames = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"]
+
+fetch("https://api.openweathermap.org/data/2.5/weather?q=Stockholm,Sweden&units=metric&APPID=f4893e301384d5deeafa555ccaa61aaa")
+.then((response) => {
+  return response.json()
+})
+.then((json) => {
+  console.log(json)
+  weatherContainer.innerHTML = `<h1> ${json.name} </h1>`
+  tempContainer.innerHTML = `<p>${json.main.temp.toFixed(1)} &#176;</p><p>${json.weather[0].description}</p>`
+
+  //use to test different weatherbackgrounds
+  //json.weather[0].description="mist"


Delete comment

________________________________

In script.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866527&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098948051&sdata=cwkRWRTJsf7tiPk18ZyO5gLuHJcYY8kuE43OHjHqwdM%3D&reserved=0>:

> @@ -0,0 +1,75 @@
+const weatherContainer = document.getElementById("weather")
+const tempContainer = document.getElementById("temp")
+const sunriseContainer = document.getElementById("sunrise")
+const sunsetContainer = document.getElementById("sunset")
+const forcastContainer = document.getElementById("forcast")
+const dayNames = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"]
+
+fetch("https://api.openweathermap.org/data/2.5/weather?q=Stockholm,Sweden&units=metric&APPID=f4893e301384d5deeafa555ccaa61aaa")
+.then((response) => {
+  return response.json()
+})
+.then((json) => {
+  console.log(json)
+  weatherContainer.innerHTML = `<h1> ${json.name} </h1>`
+  tempContainer.innerHTML = `<p>${json.main.temp.toFixed(1)} &#176;</p><p>${json.weather[0].description}</p>`


Nice of you using toFixed()!

________________________________

In script.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866553&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098948051&sdata=kTs9nR6Uy1qF%2BFcfKuklEQibTzODsP17Z4lzT%2BVtm1A%3D&reserved=0>:

> +  let sunrise = new Date(json.sys.sunrise * 1000)
+  let sunriseTime = sunrise.toLocaleTimeString([], { timeStyle: 'short' })
+
+  let sunset = new Date(json.sys.sunset * 1000)
+  let sunsetTime = sunset.toLocaleTimeString([], { timeStyle: 'short' })
+
+  sunriseContainer.innerHTML = `<p>Sunrise ${sunriseTime} </p>`
+  sunsetContainer.innerHTML = `<p>Sunset ${sunsetTime} </p>`
+})
+
+fetch("https://api.openweathermap.org/data/2.5/forecast?q=Stockholm,Sweden&units=metric&APPID=f4893e301384d5deeafa555ccaa61aaa")
+.then((response) => {
+  return response.json()
+})
+.then((json) =>{
+  console.log(json)


Delete log

________________________________

In script.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866558&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098958059&sdata=xwy10ZpVo0dtv%2BjghQMEr9Y7EdbutcTv1cg%2BrFLb2D0%3D&reserved=0>:

> +  let sunsetTime = sunset.toLocaleTimeString([], { timeStyle: 'short' })
+
+  sunriseContainer.innerHTML = `<p>Sunrise ${sunriseTime} </p>`
+  sunsetContainer.innerHTML = `<p>Sunset ${sunsetTime} </p>`
+})
+
+fetch("https://api.openweathermap.org/data/2.5/forecast?q=Stockholm,Sweden&units=metric&APPID=f4893e301384d5deeafa555ccaa61aaa")
+.then((response) => {
+  return response.json()
+})
+.then((json) =>{
+  console.log(json)
+  let forcast = ""
+  json.list.forEach((element) =>{
+    let weatherDate = new Date(element.dt * 1000)
+    console.log(weatherDate.getHours())


Delete log

________________________________

In script.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866572&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098958059&sdata=gDy2h5aO4ByZINgFB4DyLe5wqJDeBpAQskTwzUt9yXk%3D&reserved=0>:

> +
+fetch("https://api.openweathermap.org/data/2.5/forecast?q=Stockholm,Sweden&units=metric&APPID=f4893e301384d5deeafa555ccaa61aaa")
+.then((response) => {
+  return response.json()
+})
+.then((json) =>{
+  console.log(json)
+  let forcast = ""
+  json.list.forEach((element) =>{
+    let weatherDate = new Date(element.dt * 1000)
+    console.log(weatherDate.getHours())
+    if (weatherDate.getHours() === 13){
+      forcast += `<div class="forcastDay">${dayNames[weatherDate.getDay()]}:</div>`
+      forcast += `<div class="forcastTemp">${element.main.temp.toFixed(1)} &#176;</div>`
+      forcast += `<div class="forcastDescription">${element.weather[0].description}</div>`
+      console.log(dayNames[weatherDate.getDay()])


Delete log

________________________________

In script.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866580&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098968068&sdata=rTsFhoZT12hm6gtmVGE9A%2BnobAV4J%2Fu4cX7HbyXDSbg%3D&reserved=0>:

> +// container2.innerHTML = `<p>Wed: ${json.list[8].main.temp.toFixed(1)} &#176; ${json.list[8].weather[0].description}</p> <p> Thu: ${json.list[16].main.temp.toFixed(1)} &#176; ${json.list[16
+// ].weather[0].description}</p> <p> Fri: ${json.list[24].main.temp.toFixed(1)} &#176; ${json.list[24].weather[0].description}</p> <p> Sat: ${json.list[32].main.temp.toFixed(1)} &#176; ${json.list[32].weather[0].description}</p>`


Delete comments

________________________________

In script.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866587&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098968068&sdata=7G%2F0vTcJOjxJ7ksk1gGWJiOgOBFq%2Bi8kbHhmT1ff4%2Fo%3D&reserved=0>:

> +
+
+
+


Delete empty rows

________________________________

In style.css<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866681&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098978072&sdata=i20%2B%2Fj0rIJpjUfcr7kVBkrvVpx5xFWzUzAXCAi%2FNR9o%3D&reserved=0>:

> @@ -0,0 +1,107 @@
+body{
+    margin:0px;
+}
+
+* {
+    font-family: 'McLaren', cursive;
+}
+
+.backgroundContainer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: left;
+    background-size: cover;


I figure that you want the background to all over the page with this? And it's how you use it. The problem is that your body hasn't to much height. You can change that by adding height: 100vh; vh stands for viewport height

________________________________

In style.css<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866687&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098978072&sdata=2BTZJ8XRyc6aCfixVzTfso2wZ4LEi%2F3NYe%2FzTRRo3is%3D&reserved=0>:

> @@ -0,0 +1,107 @@
+body{


⬇️ Suggested change

-body{
+body {


________________________________

In style.css<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866695&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098988076&sdata=a1V7%2F8CwWT6iyA6KJIR%2FOBGL7qydSZ5wjt75HSiqIco%3D&reserved=0>:

> @@ -0,0 +1,107 @@
+body{
+    margin:0px;


⬇️ Suggested change

-    margin:0px;
+    margin: 0px;


________________________________

In style.css<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866704&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098988076&sdata=th%2BSkMsy7hXFeB5%2BjcqNwKtSc67NDrHttU4y9zuWqIY%3D&reserved=0>:

> +    background-image: linear-gradient(rgb(101, 179, 243), rgb(185, 201, 206));
+}
+
+.allWeather{
+    border: 2px solid black;
+    height: 450px;
+    margin-top: 50px;
+    margin-bottom: 50px;
+    background-color: white;
+    padding: 30px;
+    text-align: center;
+    border-radius: 20px;
+    width: 70%;
+}
+
+h1{


⬇️ Suggested change

-h1{
+h1 {


________________________________

In style.css<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-weather-app%2Fpull%2F11%23discussion_r341866715&data=02%7C01%7C%7C7efdfc68059b4219d14008d7609171a3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637084050098998084&sdata=ct6Hi%2BtN45kY%2BSYbwSit9cURdaEGwLMEM8zeP1%2BSbmQ%3D&reserved=0>:

> +    margin-top: 50px;
+    margin-bottom: 50px;


Collide these two to one row margin: 50px 0;

**Pull request closed without merging**